invenio_app_rdm/theme/assets/js/invenio_app_rdm/search/SearchMain/SearchResults/SearchResults.js

```diff
@@ -79,10 +79,18 @@ class Results extends Component {
 }
 
 const OnResults = withState(Results);
+const OnEmptyResults = () => (
```

Comment:
Created this because by default `react-searchkit/EmptyResults` component is exposing a button to reset the query. Although this works and clears the query from the url parameter, the searchbar value is not updated because there is no way currently to communicate the change from the searchkit app to the searchbar one. So, I just override the render and removed the button so people don't end up to this situation.

Reply:
Got it